experiments/ClimaEarth/components/land/climaland_integrated.jl

```diff
@@ -0,0 +1,540 @@
+import ClimaParams
+import ClimaLand as CL
+import Dates
+import ClimaUtilities.TimeVaryingInputs: LinearInterpolation, PeriodicCalendar, TimeVaryingInput
+import ClimaCoupler: Checkpointer, FieldExchanger, FluxCalculator, Interfacer, Utilities
+import ClimaCore as CC
+import SciMLBase
+import ClimaTimeSteppers as CTS
+import ClimaDiagnostics as CD
+import ClimaUtilities.TimeManager: ITime
+
+include("climaland_helpers.jl")
+
+"""
+    ClimaLandSimulation{M, P, D, I, A}
+
+The integrated ClimaLand model simulation object.
+"""
+struct ClimaLandSimulation{M, D, I, A} <: Interfacer.LandModelSimulation
+    model::M
+    domain::D
+    integrator::I
+    area_fraction::A
+end
+
+"""
+    ClimaLandSimulation(
+        ::Type{FT},
+        dt::TT,
+        tspan::Tuple{TT, TT},
+        start_date::Dates.DateTime,
+        output_dir::String,
+        boundary_space,
+        area_fraction;
+        saveat::Vector{TT} = [tspan[1], tspan[2]],
+        domain_type::String = "sphere",
+        land_temperature_anomaly::String = "amip",
+        use_land_diagnostics::Bool = true,
+        stepper = CTS.ARS111(),
+    )
+
+Creates a ClimaLandSimulation object containing a land domain,
+a ClimaLand.LandModel, and an integrator.
+
+This type of model contains a canopy model, soil model, snow model, and
+soil CO2 model. Specific details about the complexity of the model
+can be found in the ClimaLand.jl documentation.
+"""
+function ClimaLandSimulation(
+    ::Type{FT},
+    dt::TT,
+    tspan::Tuple{TT, TT},
+    start_date::Dates.DateTime,
+    output_dir::String,
+    boundary_space,
+    area_fraction;
+    saveat::Vector{TT} = [tspan[1], tspan[2]],
+    domain_type::String = "sphere",
+    surface_elevation = CC.Fields.zeros(boundary_space),
+    land_temperature_anomaly::String = "amip",
+    use_land_diagnostics::Bool = true,
+    stepper = CTS.ARS111(),
```

Comment:
is this the first time we use an implicit stepper for a surface model? just curious.